Runtime/CourseController/Controllers/BaseCourseController.cs

```diff
@@ -1,20 +1,24 @@
 ﻿using System;
 using System.Collections.Generic;
+using Innoactive.Creator.Core.Internationalization;
 using UnityEngine;
 
 namespace Innoactive.Creator.UX
 {
     /// <summary>
     /// Base course controller which instantiates a defined prefab.
     /// </summary>
-    public abstract class BaseCourseController : ICourseController
+    public abstract class BaseCourseController : ICourseController, ILocalizationProvider
```

Comment:
I have the feeling we are creating a monster with the CourseController :D but it probably needs a full rework at some point then

Reply:
Naaa... course controller also decides which mode is used and depending on the mode it should also decided which language configuration is loaded.